test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -133,6 +133,56 @@ test.describe(
 				)
 			).toBeVisible();
 		} );
+
+		test( 'default to my most recently created menu', async ( {
+			admin,
+			page,
+			editor,
+			requestUtils,
+		} ) => {
+			await admin.createNewPost();
+			await requestUtils.createNavigationMenu( {
+				title: 'Test Menu 1',
+				content:
+					'<!-- wp:navigation-link {"label":"WordPress","type":"custom","url":"http://www.wordpress.org/","kind":"custom","isTopLevelLink":true} /-->',
```

Comment:
Similar to my other comment — Changing this from "WordPress" to "Menu 1 Link" or "Old Menu Link" that makes it clear that it should _not_ be showing in our artifacts would be helpful for debugging.